build-system/tasks/visual-diff.rb

```diff
@@ -134,9 +136,20 @@ def generateSnapshots(pagesToSnapshot)
 # - page: Page object used by Percy for snapshotting.
 # - url: Relative URL of page to be snapshotted.
 # - name: Name of snapshot on Percy.
-def generateSnapshot(page, url, name)
+# - required_css: Array of CSS elements that must be found in the page.
```

Comment:
Lets make the names more explicit like `loaded_when_css_present` and `not_loaded_while_css_present`